net-mail/swaks/swaks-20201010.0.ebuild

```diff
@@ -0,0 +1,48 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit perl-functions
+
+DESCRIPTION="Swiss Army Knife SMTP; Command line SMTP testing, including TLS and AUTH"
+HOMEPAGE="http://www.jetmore.org/john/code/swaks"
+SRC_URI="http://www.jetmore.org/john/code/swaks/${P}.tar.gz"
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86"
+
+IUSE="ssl"
```

Comment:
This doesn't seem to have any function in this ebuild, move it to `optfeature` instead.
https://projects.gentoo.org/qa/policy-guide/dependencies.html#pg0001